src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/paragraph/helpers.js

```diff
@@ -31,6 +31,8 @@ export const processBlock = _block => {
     if (block.role === 'introduction') {
       block.text = boldWrap(block.text);
       block.markupType = 'candy_xml';
+    } else if (block.text.includes('</link>')) {
```

Comment:
Asked on slack but should post it here too - this PR only updates paragraphs - does this cover everything?  Seen the bug in bulleted lists too

Reply:
Fixes it in the bulleted list in the example - http://localhost:7080/mundo/video_fotos/2016/05/160502_video_apnea_record_rubridge_nezolandes_amv

Looks like the list preprocessor imports `convertParagraph` and runs this list items through it.  `convertParagraph` runs the block through `processBlock`, which is the helper that captures is logic.